### PR TITLE
35 - resolve #6

### LINKED ISSUE #6: Meetup : 35 | Planner
# Event Details

**Meetup No** : 35
**Proposed date**  : August 16th
**Mode** : Offline
**Confirmed date** :  August 16th

**Meetup:**         https://www.meetup.com/DevOps-Kathmandu/events/
**Meetup Link:** https://www.meetup.com/devops-kathmandu/events/302670562

## Speaker Bio
**Speaker 1: Sudeep Bhandari**
Software Engineer / DevOps Engineer
Linkedin: [https://www.linkedin.com/in/sudeephb/](https://www.linkedin.com/in/sudeephb/)
Github: [https://github.com/sudeephb](https://github.com/sudeephb)

---

**Speaker 2: Milan Thapa** 
Ops guy aspiring to be dev
Website: [https://thapakazi.com/](https://thapakazi.com/)
Twitter: [https://x.com/thapakazi_](https://x.com/thapakazi_)

## Meetup Description

**Title:**  35th Meetup | Creating software to Monitor Hardware
**Description:** We warmly invite you to attend the 35th episode of DevOps Kathmandu in person meetup, featuring engaging discussions on Creating Software to Monitor Hardware, Simplify Your Docker Workflow with Lazydocker and awesome interaction with fellow DevOps.

## Resources
**Curation:**       https://github.com/DevOps-Kathmandu/curation
**Youtube:**       https://www.youtube.com/channel/UChIow3i-WObHKidPD_xwwlA/videos
**Slack:**           http://bit.ly/devops-kathmandu-slack
**Q/A docs:**     http://bit.ly/devops-kathmandu-area.
 
## Actions / Flow
Step 1(Pre plan)
- [x] Choose a lead organizer for catching up the flow and at least two organizers
- [x] Create a centralized google doc for the meetup inside community google drive for writing updates
- [x] Publish for seeking speakers and sponsors
- [x] Confirm date and time with speaker
- [x] Confirm venue(Requirement: wireless mic and speaker, max 60, spacious hall, water-tea-coffee-cookies, parking space, food court)
- [x] Confirm parking in venue for 2/4 wheels
- [x] Finalize sponsor
- [x] Finalize speaker
- [x] Collect speakers { bio, meetup title, description, time limit, target audience, photo, phone number for emergency updates(cancellation, time change, etc.) in centralized google doc }

Step 2(Ready for it!)
- [x] Media banner with speaker photo
- [x] Publish to meetup group and make announcement at least 10 days before event
- [x] Inform speaker about publication of event and send link along with RSVP summary. Also, send google calendar invite with meet link to use in meetup day for screenshare recording
- [x] Announce to email, linkedin, twitter, slack, discord
- [x] Reminder message to attendees T-1(Include time, venue map & routes, parking options)

Step 3(Meetup Day)
- [x] Paste venue direction and meetup info in A4 sheets if required
- [x] Manage water, tea, coffee, cup, chairs
- [x] Mic, speaker, projector test
- [x] Start screenshare of speaker via pre-generated meet url 
- [x] Update Q/A docs(for online meetup)

Step 4(Post Meetup)
- [x] Send thank you note to speakers and host. Also, ask speakers to add slides on community google drive folder
- [x] Confirm slides from speakers
- [x] Send thanks and review forms to attendees via meetup. Include slides and videos if available
- [ ] Do a retro and share outcomes in this issue

### PR DESCRIPTION
**Pull request by thapakazi**
resolve #6 

**TODO**

- [x]  need link from **Sudeep Bhandari**

### CODE REVIEW

File: 2024/20240816-#35/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# DevOps Kathmandu Meetup: 35th Ed.
+
+## Date and Time
+16th October, 2024 - Friday, 17:00 to 19:00 NPT
+
+## Talks and Speakers
+- "Creating software to Monitor Hardware" - [Sudeep Bhandari](https://www.linkedin.com/in/sudeephb/)
+  - [Slides](....)
```
Comment (thapakazi) [inline 2024/20240816-#35/README.md:8]: need slide link from **Sudeep Bhandari**

Comment (dwdraju) [reply 2024/20240816-#35/README.md:8]: @sudeephb Can you please share slide link?

Comment (sudeephb) [reply 2024/20240816-#35/README.md:8]: Please find the slide link [here](https://docs.google.com/presentation/d/1_4rP00l1VnJiwRGvy2AeNLMsR7Pra_jt04tMsQ1iI9I/edit?usp=sharing).

**Review by omegazyadav — APPROVED** [review]
LGTM

**Review by dwdraju — APPROVED** [review]

**Pull request merged**